docs/advanced-curriculum.adoc

```diff
@@ -3,12 +3,12 @@
 include::config/setup.adoc[]
 
 ifeval::["{language}" == "DE"]
-= Curriculum fürpass:q[<br><br>]Certified Professional forpass:q[<br>]Software Architecture (CPSA)^®^pass:q[<br>]_Advanced Level_: pass:q[<br>] Modulpass:q[<br>]{curriculum-short}pass:q[<br><br>]{curriculum-name}
+= Curriculum fürpass:q[<br><br>]Certified Professional forpass:q[<br>]Software Architecture (CPSA)^(R)^pass:q[<br>]_Advanced Level_: pass:q[<br>] Modulpass:q[<br>]{curriculum-short}pass:q[<br><br>]{curriculum-name}
```

Comment:
The position of the (registered) symbol is not consistent! Sometimes it's on the term "Certified Professional for Software Architecture" and sometimes after the "CPSA" abbreviation. @gernotstarke do you know what the right position is?

Reply:
no - but imho that's not critical. Let's:

-  put it behind CPSA, 
- always use superscript 
- always use AsciiDoc macro instead of unicode character

-> right way to do it: CPSA^(R)^